tests/authentication.rs

```diff
@@ -35,7 +35,7 @@ mod authentication_tests {
     /// Call out to cargo to start trow.
     /// Seriously considering moving to docker run.
 
-    fn start_trow() -> TrowInstance {
+    async fn start_trow() -> TrowInstance {
```

Comment:
I'm not sure it's worth moving all the test code to async hyper. Isn't making it more complicated?

Reply:
We are using a more recent version of `reqwest` which is async by default. So whenever we use reqwest now we need to make the function async.  
`start_trow` is using `reqwest` to verify that Trow has started. This is why I have to use `async` everywhere.

Reply:
I think you can still turn it on by enabling the "blocking" feature. Maybe it's not worth it though.